crates/turborepo-ui/src/tui/app.rs

```diff
@@ -201,6 +201,15 @@ impl<W> App<W> {
         Ok(())
     }
 
+    pub fn scroll_terminal_output_by_page(&mut self, direction: Direction) -> Result<(), Error> {
+        let pane_rows = self.size.pane_rows();
+        let task = self.get_full_task_mut()?;
+        // Scroll by the height of the terminal pane
+        task.scroll_by(direction, usize::from(pane_rows))?;
```

Comment:
I hope you don't mind, I added a new method that generalized scrolling instead of scrolling by one in a for loop

Reply:
Nice!